src/USER-FEP/pair_lj_class2_soft.h

```diff
@@ -0,0 +1,69 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef PAIR_CLASS
+
+PairStyle(lj/class2/soft,PairLJClass2Soft)
+
+#else
+
+#ifndef LMP_PAIR_LJ_CLASS2_SOFT_H
+#define LMP_PAIR_LJ_CLASS2_SOFT_H
+
+#include "pair.h"
+
+namespace LAMMPS_NS {
+
+class PairLJClass2Soft : public Pair {
+ public:
+  PairLJClass2Soft(class LAMMPS *);
+  virtual ~PairLJClass2Soft();
+  virtual void compute(int, int);
+  virtual void settings(int, char **);
+  void coeff(int, char **);
+  virtual double init_one(int, int);
+  void write_restart(FILE *);
+  void read_restart(FILE *);
+  virtual void write_restart_settings(FILE *);
+  virtual void read_restart_settings(FILE *);
+  void write_data(FILE *);
+  void write_data_all(FILE *);
+  virtual double single(int, int, int, int, double, double, double, double &);
+  void *extract(const char *, int &);
+
+ protected:
+  double cut_global;
+  double **cut;
+  double **epsilon, **sigma, **lambda;
+  double nlambda, alphalj;
+  double **lj1,**lj2,**lj3,**offset;
+
+  virtual void allocate();
+};
+
+}
+
+#endif
+#endif
+
+/* ERROR/WARNING messages:
+E: Illegal ... command
+Self-explanatory.  Check the input script syntax and compare to the
+documentation for the command.  You can use -echo screen as a
+command-line option when running LAMMPS to see the offending line.
+
+E: Incorrect args for pair coefficients
+Self-explanatory.  Check the input script or data file.
+
+E: Pair lj/cut/soft different lambda values in mix
+The value of lambda has to be the same for I J pairs.
+
+*/
```

Comment:
No newline at end of file.

Reply:
fixed in ae6fb92478c0c7373967cc47c21d4667894fa8f8